src/plugins/base64/README.md

```diff
@@ -41,4 +41,24 @@ sudo kdb umount /examples/base64/test
 
     @BASE64SGVsbG8gV29ybGQhCg==
 
-.
+. The following example shows how you can use this plugin together with INI to store binary data.
+
+```sh
+# Mount the INI and Base64 plugin
+kdb mount config.ini user/examples/base64 ini base64
+
+# Copy binary data
+kdb cp system/elektra/modules/dump/exports/unserialise user/examples/base64/binary
+
+# Print binary data
+kdb get user/examples/base64/binary
+# STDOUT-REGEX: (\\\\x[0-9a-f]+)+
```

Comment:
does this make sure that *only* `[0-9a-f]` are present? (is the regex implicitely surrounded with `^$`?)

Reply:
Nope. I replaced the regex with the more rigorous `^(\\\\x[0-9a-f]{1,2})+$`.